### DISCUSSION TITLE
Relations are put 'inline'

### DISCUSSION THREAD

**Discussion by mariusgerome** [RafaelGB/obsidian-db-folder#990] [Q&A]
Database: "LIBRARY" (books)
Field: `author` ; property: `relation` ; relates to: "People Database"

If I relate "Moby Dick" to "Herman Melville" the field `author::` is placed inline, like so:

```
---
type: book
---
# Moby Dick
author:: [[Herman Melville]]
```

I expected it to be placed inside the yaml frontmatter.
Is there a way to change this? Haven't found a setting for that.

– Marius

**Comment by Youaresmarter** [comment]
The same question from 2024.
